### ISSUE TITLE
tensorboard not showing data

### ISSUE THREAD

**Issue by snoopypi** [tensorflow/tensorboard#3107]
Hi there
I trained several times and now have a lot of events.out.tfevents- files.
Now I would like to analyze these data in tensorboard because I need the accuracy/mAP of the different training sessions.

Problem now if I open tensorboard with my last training session:
tensorboard --logdir="./traing"

I only see learning rate and a (strange) Loss. (see printscreen)

How can i see more? Especially the accuracy and so?

I have tensorflow/board 1.13.1, Ubuntu 18.04

I followed this tutorial:
https://tensorflow-object-detection-api-tutorial.readthedocs.io/en/latest/training.html

Second screenshots show the files from one trainining folder.

Thank you
![Screenshot from 2020-01-04 10-53-34](https://user-images.githubusercontent.com/48096442/71777267-d188dd00-2f9d-11ea-9a3e-8f44e49c521f.png)
![Screenshot from 2020-01-04 11-02-36](https://user-images.githubusercontent.com/48096442/71777269-d77ebe00-2f9d-11ea-9346-b592c02bb46f.png)

**Comment by rmothukuru** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow-tensorboard) since it is not a bug or feature request. There is also a larger community that reads questions there.

**Comment by psybuzz** [comment]
Thanks for the question!  To what was said, StackOverflow is usually a better place to get community answers for this type of thing.

At first glance, the tutorial you linked mentions downloading models from https://github.com/tensorflow/models, and using the training scripts provided within.  Code inside [trainer.py](https://github.com/tensorflow/models/blob/master/research/object_detection/legacy/trainer.py#L359), and [losses.py](https://github.com/tensorflow/models/blob/fe748d4a4a1576b57c279014ac0ceb47344399c4/research/object_detection/core/losses.py#L630), are writing the scalars shown in your screenshot, using `tf.summary.scalar`.

You may want to add another `tf.summary.scalar(accuracy)` call for metric you need, or ask the maintainer of those scripts to do so.

**Comment by snoopypi** [comment]
Hi thank you very much. The problem is that I already did the training. I do not have the time to train all models again.. I just have the eventfiles of the different trainings.. :-/

**Comment by psybuzz** [comment]
I see, re-training would be unfortunate.  If the metrics you want can somehow be computed from the logged metrics that already exist within the event files, then one option would be to download the scalar data as CSV or JSON from TensorBoard's UI, and work with that.

https://stackoverflow.com/questions/42355122/can-i-export-a-tensorflow-summary-to-csv

**Comment by snoopypi** [comment]
> I see, re-training would be unfortunate. If the metrics you want can somehow be computed from the logged metrics that already exist within the event files, then one option would be to download the scalar data as CSV or JSON from TensorBoard's UI, and work with that.
> 
> https://stackoverflow.com/questions/42355122/can-i-export-a-tensorflow-summary-to-csv

Hi there thanks for your reply.
Well I managed now that eval.py show mAP. But its always 1.0!!  Thats not possible..

Do you know where the bug might be?
